diesel/Cargo.toml

```diff
@@ -39,7 +40,7 @@ unstable = []
 lint = ["clippy"]
 large-tables = []
 huge-tables = ["large-tables"]
-postgres = ["pq-sys"]
+postgres = ["pq-sys", "ipnetwork"]
```

Comment:
If you really need the external dependency, please use it as a feature flag itself instead of adding in to the postgres feature (not everyone will need to use these types).